docs/language/learn-ql/about-ql.rst

```diff
@@ -1,11 +1,11 @@
 About QL
```

Comment:
It seemed to me that this topic was mostly about the language, so I mostly left terminology as-is.